changelog/15428.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+login/mfa: Fixed erroneous success message in case of two-phase MFA, and fixed MFA information missing from table format.
```

Comment:
Would you think just "auth" be better as the category? I think we went with that before, worth checking it out. 
Also, I guess it would be nice to note the commands in question in the message.

Reply:
I went with login/mfa to try and make it clear that it was the non-enterprise MFA - but I'm open to changing it. I couldn't find any PR that touched a similar area but had a changelog.

Would you suggest auth/mfa, login/auth, or just auth? 

You're right about including the command in the log - I'll update that!

Reply:
I would recommend to go with just "auth". The ENT MFA does not have the concept of two phase login, so that part of the message shows which MFA system is fixed.

Reply:
Updated